Traceback (most recent call last):
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/anaconda3/envs/vlxenv/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
molecule.set_multiplicity(3)
space=mtp.MolSpace(molecule,scfdrv.mol_orbs)
space.CAS(8,6)

mcscfdrv=mtp.McscfDriver(molecule,basis,space)
mcscfdrv.compute(1)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
[0;32m/var/folders/y6/hqkhhjrd3ngdjcp8bf70_hy80000gn/T/ipykernel_1267/3514664834.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0mmolecule[0m[0;34m.[0m[0mset_multiplicity[0m[0;34m([0m[0;36m3[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mspace[0m[0;34m=[0m[0mmtp[0m[0;34m.[0m[0mMolSpace[0m[0;34m([0m[0mmolecule[0m[0;34m,[0m[0mscfdrv[0m[0;34m.[0m[0mmol_orbs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0mspace[0m[0;34m.[0m[0mCAS[0m[0;34m([0m[0;36m8[0m[0;34m,[0m[0;36m6[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0;34m[0m[0m
[1;32m      5[0m [0mmcscfdrv[0m[0;34m=[0m[0mmtp[0m[0;34m.[0m[0mMcscfDriver[0m[0;34m([0m[0mmolecule[0m[0;34m,[0m[0mbasis[0m[0;34m,[0m[0mspace[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mAttributeError[0m: module 'multipsi' has no attribute 'MolSpace'
AttributeError: module 'multipsi' has no attribute 'MolSpace'

